index.bs

```diff
@@ -180,6 +180,26 @@ by showing how many sites are willing to ask for intrusive and unnecessary permi
 
 <h2 id="api-across-languages">API Design Across Languages</h2>
 
+<h3 id="simplicity">Prefer simple solutions</h3>
+<!-- was "Avoid Needless Complexity" in the HTML Design Principles -->
```

Comment:
I'm not sure I like this rewording, but perhaps it can be merged with "baby steps" somehow?

Reply:
@annevk I think they're distinct design principles, but if you'd like I can add my import of Baby Steps to this PR and we can see what we think then.

Reply:
Actually, I think I'd rather do Baby Steps in a separate, follow-on PR. We cane valuate then if they should be merged.